react/week4/github_user_search/src/consumeUser.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useEffect } from "react";
+import { UserSearchContext } from "./userSearch.js";
+
+const Api_url = "https://api.github.com/search/users?q=";
+
+const User = () => {
+  const userContext = useContext(UserSearchContext);
+  console.log(userContext);
```

Comment:
Remember to remove debugging commands :)

Reply:
Ohh it means console??, I will remove now, sry.